data/human/human missions.txt

```diff
@@ -4077,3 +4077,135 @@ mission "Laborer Payment"
 				decline
+
+
+mission "Riley"
+	name "Visiting Home"
+	description "Take a woman named Riley to see her ailing father on <destination> before it's too late."
+	minor
+	passengers 1
+	source
+		attributes deep 
+		attributes urban
+	deadline
+	destination "Memory"
+	to offer
+		random < 15
+	on offer
+		conversation
+			`Entering the spaceport, you notice a young woman moving from captain to captain, probably looking to hire one.`
+			choice
+				`	(Ask the young woman if you can help her.)`
+				`	(Ignore her.)`
+					decline
+			`	You approach the woman and ask her if she is looking to hire a pilot.`
+			`	"I need to get to <destination> to see my father. He's dying. I have <payment> credits to pay for the journey; please, will you help me? I need to be there in <deadline> days, at the most."`
+			`	Her eyelids are baggy, her clothes wrinkled, and she holds herself rigidly; signs of desperation, stress, and sleep deprivation.`
+			choice
+				`	"I can help you. We'll make haste to reach <planet> in time."`
+					goto help
+				`	"I don't have time to take on a passenger right now."`
+			`	She nods, and moves on to another captain to beseech instead.`
+				decline
+			label help
+			`	Her eyes widen, and she opens her mouth twice before words come out. "Thank you! This means the world to me. My name is Riley, by the way. I don't have much luggage; I'll be ready to leave as soon as you are."`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Riley hasn't entered this system yet. Better depart and wait for it.`
+	on complete
+		payment 13000
+		conversation
+			`In the days since Riley came aboard, she has relaxed and cleaned up. She looks nearly unrecognizable from the tired, desperate woman you first saw.`
+			`	On the way, she told you stories of her past, mostly of time shared with her father. As a young teen, they spent summers riding the latest horse mechs through the forest, looking for beehives to steal honey from. At age 19, she broke her ankle hoverboarding on a restricted mountain peak. Her father rescued her, avoiding the police to hike up with a medical kit. She talks about his wild birthday gifts to her each year, always a different piece of clothing with embedded AI that was just shy of what was allowed by law. As you watch, she shows how her shirt warms and constricts if it detects stress, mimicking a hug.`
+			`	She didn't mention it, but it's clear that she was concerned that she'd arrive too late to say goodbye.`
+			`	As you neared <system>, Riley told you about the last time she saw her father. He had become focused on his work - research into safe AI - and they hadn't seen each other for years. She had become a well-known DJ back on <source> and had a break in her touring schedule, so she made contact and they met halfway. They spent most of a week together, reliving old times.`
```

Comment:
"Meeting halfway" seems a bit weird when Deep space is pretty small. You could even pick this mission up a couple jumps away at Valhalla. Having her visiting on Memory would probably be the simplest, although it'd beg the question of why she's having trouble getting there now. Up to you if you want to fix it.

Reply:
Tried fixing it, is it ok now?

Reply:
Yep, looks good!